### ISSUE TITLE
AttributeError: 'module' object has no attribute 'get_terminal_size'

### ISSUE THREAD

**Issue by AlphaHeX** [Wintermute0110/torrentverify#2]
I'm getting:
Traceback (most recent call last):
  File "torrentverify.py", line 59, in <module>
    __cols, __lines = shutil.get_terminal_size()
AttributeError: 'module' object has no attribute 'get_terminal_size'

I have python 3.2 and python 2.7 installed (default is python3)

**Comment by Wintermute0110** [comment]
Hi,

please tell me your Python 3 version. Run,

`$ python3 --version`

Also tell me which platform you are using (Linux, Windows Cygwin, etc.).

**Comment by AlphaHeX** [comment]
Hi,

I'm running Linux.

python3 --version -> Python 3.2.3

Regards,

**Comment by Wintermute0110** [comment]
OK, thanks.

torrentverify relies on [shutils.get_terminal_size()](https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size) to learn about the size of the terminal, so text lines that exceed that length are chopped off. shutils.get_terminal_size() is only available after Python 3.3, and hence the error you get when you use Python 3.2.3.

Solution: I will check if Python version is 3.3 or greater. If not, chopping of long lines of text will be disabled. User will be warned yo upgrade to Python >= 3.3. Torrentverify 0.1.1 will have the fix for this problem.

**Comment by AlphaHeX** [comment]
```
                                                                                  Thank you                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        Od: Wintermute 0110Wysłano: czwartek, 11 lutego 2016 09:25Do: Wintermute0110/torrentverifyOdpowiedz: Wintermute0110/torrentverifyqQQ‎DW: AlphaHeXTemat: Re: [torrentverify] AttributeError: 'module' object has no attribute 'get_terminal_size' (#2)OK, thanks.
```

torrentverify relies on shutils.get_terminal_size() [1] to learn about the size of the terminal, so text lines that exceed that length are chopped off. shutils.get_terminal_size() is only available after Python 3.3, and hence the error you get when you use Python 3.2.3.

Solution: I will check if Python version is 3.3 or greater. If not, chopping of long lines of text will be disabled. User will be warned yo upgrade to Python >= 3.3. Torrentverify 0.1.1 will have the fix for this problem.

[1] https://docs.python.org/3/library/shutil.html#shutil.get_terminal_size

—Reply to this email directly or view it on GitHub.
